config.js

```diff
@@ -254,6 +258,9 @@ module.exports = {
   getSamplesWildcardCacheInvalidation:
     pe.GET_SAMPLES_WILDCARD_CACHE_INVALIDATION ||
     DEFAULT_GET_SAMPLES_WILDCARD_CACHE_INVALIDATION,
+  missedCollectorHeartbeatCheckIntervalMillis:
+    +pe.MISSED_COLLECTOR_HEARTBEAT_CHECK_INTERVAL_MILLIS ||
```

Comment:
I don't think you need to define a new interval variable for this, see heartbeatLatencyToleranceMillis in ./config/collectorConfig.js

Reply:
@iamigo Sorry didn't realize this PR is still open with a comment - So do we want to check for missed heartbeats every heartbeatLatencyToleranceMillis (or heartbeatIntervalMillis?) ?
In the story it said - "use config var: MISSED_COLLECTOR_HEARTBEAT_CHECK_INTERVAL_MILLIS. default 50000ms"
fyi @jgraff2

Reply:
I think "heartbeatLatencyToleranceMillis" makes sense, based on the comments associated with that variable in /config/collectorConfig.js

Reply:
okay, let me update the PR.